parm/config/config.waveprep

```diff
@@ -20,20 +20,24 @@ export POFILETYPE=1     # POFILETYPE=1 one point file per output step
 # Unified output T or F
 export FUNIPNT='T'
 # Output server type (see ww3_multi.inp in WW3 repo)
-export IOSRV='3'
+export IOSRV='1'
 # Flag for dedicated output process for unified points
 export FPNTPROC='T'
 # Flag for grids sharing dedicated output processes
 export FGRDPROC='F'
+# Flag for masking computation in two-way nesting
+export FLAGMASKCOMP="F"
+# Flag for masking at printout time.
+export FLAGMASKOUT="F"
 
 # Wind interval for standalone file-based runs
 # Output stride
 export WAV_WND_HOUR_INC=1      # This value should match with the one used in
                          # the wind update script
 # Intake currents settings
 export WAV_CUR_DT=${WAV_CUR_DT:-3}
-export WAV_CUR_HF_DT=${WAV_CUR_HF_DT:-3} # Defaults to 3h for GEFSv12
-export WAV_CUR_HF_FH=${WAV_CUR_HF_FH:-0} # Constant DT for GFSv16 from getgo
+export WAV_CUR_HF_DT=${WAV_CUR_HF_DT:-1}
```

Comment:
Why is this being changed?

Reply:
The example config.wave (/work/noaa/global/wkolczyn/save/global-workflow/expdir/example/config.wave) has `export WAV_CUR_HF_DT=1`. In the develop branch we set that variable in config.waveprep so I changed it in config.waveprep to match the example config.wave setting. Let me know what it should be, thanks!

Reply:
Please revert this and the line below to what is in develop as we are using the develop/gfsv16 case

Reply:
Reverting, thanks!